src/Commands/InstallCommand.php

```diff
@@ -116,6 +118,29 @@ public function installSwooleServer()
         return true;
     }
 
+    /**
+     * Install the FrankenPHP server.
+     *
+     * @return bool
+     */
+    public function installFrankenPhpServer()
+    {
+        if (File::exists(base_path('.gitignore'))) {
+            collect(['frankenphp', 'Caddyfile', 'frankenphp-worker.php'])
+                ->each(function ($file) {
+                    $contents = File::get(base_path('.gitignore'));
+                    if (! Str::contains($contents, $file.PHP_EOL)) {
+                        File::append(
+                            base_path('.gitignore'),
+                            $file.PHP_EOL
+                        );
+                    }
+                });
+        }
+
+        return $this->ensureFrankenPhpBinaryIsInstalled();
+    }
```

Comment:
The `installFrankenPhpServer()` method works well, but I'd prefer a version with fewer file operations, even though it's not a big deal in the context of this part (installation, etc.).

My version (of course, you can completely ignore it if you want :blush: ):

```suggestion
    public function installFrankenPhpServer()
    {
        $gitIgnorePath = base_path('.gitignore');

        if (File::exists($gitIgnorePath)) {
            $contents = File::get($gitIgnorePath);

            $filesToAppend = collect(['frankenphp', 'Caddyfile', 'frankenphp-worker.php'])
                ->filter(function ($file) use ($contents) {
                    return !str_contains($contents, $file . PHP_EOL);
                })
                ->implode(PHP_EOL);

            if ($filesToAppend !== '') {
                File::append($gitIgnorePath, PHP_EOL . $filesToAppend . PHP_EOL);
            }
        }

        return $this->ensureFrankenPhpBinaryIsInstalled();
    }
```

Reply:
A couple more lines can be saved with:

```php
                ->filter(fn ($file) => !str_contains($contents, $file . PHP_EOL))
```

Reply:
> A couple more lines can be saved with:
> 
> ```
>                 ->filter(fn ($file) => !str_contains($contents, $file . PHP_EOL))
> ```

Suggestion applied. 👌

Reply:
Thanks! I must admit that I copy/pasted/adapted code from other engines but I prefer your approach too!